src/broker/Manager.cc

```diff
@@ -729,15 +729,15 @@ bool Manager::PublishLogCreate(EnumVal* stream, EnumVal* writer,
 
 	if ( ! stream_id )
 		{
-		reporter->Error("Failed to remotely log: stream %d doesn't have name", stream->AsEnum());
+		reporter->Error("Failed to remotely log: stream %lld doesn't have name", stream->AsEnum());
```

Comment:
Going to change these to "..%" PRI64 "..." as they trigger compile warnings on my end:
```
../src/broker/Manager.cc:795:54: warning: format ‘%lld’ expects argument of type ‘long long int’, but argument 3 has type ‘zeek_int_t’ {aka ‘long int’} [-Wformat=]
  795 |   reporter->Error("Failed to remotely log: writer %lld doesn't have name", writer->AsEnum());
      |                                                   ~~~^                     ~~~~~~~~~~~~~~~~
      |                                                      |                                   |
      |                                                      long long int                       zeek_int_t {aka long int}
      |                                                   %ld
```